### PR TITLE
disallow random admin registrations

### PR DESCRIPTION
**Pull request by jhayes45**
admin registration now disallowed via API.

### CODE REVIEW

File: src/api/controllers/authentication.js [hunk 1]
```diff
@@ -41,7 +41,10 @@ function register(req, res) {
                     user = new Recruiter();
                     break;
                 case 'admin':
-                    user = new Admin();
+                    return res.status(403).json({
```
Comment (brianwang9100) [inline src/api/controllers/authentication.js]: so how do we create the admin account then?

Comment (jhayes45) [reply src/api/controllers/authentication.js]: mongo shell

Comment (jhayes45) [reply src/api/controllers/authentication.js]: idk, maybe we don't need to restrict it entirely, but we need some measure in place against people registering as admin. Right now it's easier to register as an admin than as a recruiter, at least via the api

**Comment by jhayes45** [comment]
will fix later, since for testing purposes it's handy to allow

**Pull request closed without merging**